public/data/pranay202.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "Pranay Kharabe",
+    "type": "personal",
+    "bio": "I'm a full stack MERN developer and an open-source enthusiast specializing in building exceptional digital experiences. Currently focused on building responsive full stack web applications.",
+    "avatar": "https://github.com/pranay202.png",
+    "links": [
+    {
+        "name": "Connect on Linkedin",
+        "url": "https://www.linkedin.com/in/pranay-kharabe-b01456207/",
+        "icon": "linkedin"
+    },
+    {
+        "name": "Connect on GitHub",
+        "url": "https://github.com/pranay202",
+        "icon": "github"
+    },
+    {
+        "name": "Connect on Discord",
+        "url": "https://discord.com/channels/@me/KharabePranay#9791",
+        "icon": "discord"
+    },
+    {
+        "name": "Connect on Twitter",
+        "url": "https://twitter.com/kharabePranay23",
+        "icon": "twitter"
+    },
+    {
+        "name": "Connect on Instagram",
+        "url": "https://linkedin.com/in/pranay-kharabe-b01456207/",
+        "icon": "linkedin"
+    }
+    ],
+    "milestones": [
+      {
+        "title": "Started Freelancing",
+        "date": "May 2021",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started freelancing",
+        "url": "https://www.pranaykharabe.netlify.app/"
+      },
+      {
+        "title": "Earned Gold Badge for 10DaysOfJS on HackerRank",
+        "date": "July 2021",
+        "icon": "book",
+        "color": "yellow",
+        "description": "Gold badge for 10 Days of JS",
+        "url": "https://www.hackerrank.com/kharabepranay"
+      },
+      {
+        "title": "Earned Linkedin skill Badge: Javascript",
+        "date": "July 2021",
+        "icon": "linkedin",
+        "color": "yellow",
+        "description": "In the top 5% of 897.2k people who took this.",
+        "url": "https://www.linkedin.com/feed/update/urn:li:activity:6953234827391250432/"
```

Comment:
I don't think this link (and the one below) will work as it's in your feed activity.
You should probably add a direct link so that anyone can check it. 🙂